apps/api/tsconfig.json

```diff
@@ -6,5 +6,5 @@
 		"module": "NodeNext",
 		"moduleResolution": "NodeNext"
 	},
-	"include": ["src", "./drizzle.config.ts"]
+	"include": ["src", "./drizzle.config.ts", "tests"]
```

Comment:
is there a reason why we are including `test` folder in here?

Reply:
This was an auto update from moving folders around, doesn't need to be included